src/timeline/Timeline.js

```diff
@@ -0,0 +1,216 @@
+// @flow
+import {
+  View,
+  Text,
+  ScrollView,
+  TouchableOpacity,
+  Dimensions,
+} from 'react-native';
+import populateEvents from './Packer';
+import React from 'react';
+import moment from 'moment';
+import _ from 'lodash';
+import styleConstructor from './style';
+
+const LEFT_MARGIN = 60 - 1;
+// const RIGHT_MARGIN = 10
+// const CALENDER_HEIGHT = 2400;
+// const EVENT_TITLE_HEIGHT = 15
+const TEXT_LINE_HEIGHT = 17;
+// const MIN_EVENT_TITLE_WIDTH = 20
+// const EVENT_PADDING_LEFT = 4
+
+function range(from, to) {
+  return Array.from(Array(to), (_, i) => from + i);
+}
+
+let {width: dimensionWidth} = Dimensions.get('window');
+
+export default class Timeline extends React.PureComponent {
+  constructor(props) {
```

Comment:
You're not declaring any of the props and their types. This is a very confusing API

Reply:
@Inbal-Tish Done!